### ISSUE TITLE
45663655: UIView.accessibilityIgnoresInvertColors got ignored after dismissing a presented view controller

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#20638]
#### Description

Summary:
When iOS 11/12 Smart Invert feature is on, set ignoring all UIViews with UIView.appearance() protocol, present then dismiss a view controller, the presenting view controller will ignore the ignoring setting.

Steps to Reproduce:
1. On an iOS 11 or 12 device, enable Smart Invert
2. Launch the app
3. In application(didFinishLaunchingWithOptions:), call UIView.appearance().accessibilityIgnoresInvertColors = true
4. Present a view controller
5. Dismiss the view controller

Expected Results:
Presenting view controller and its subviews ignore invert.

Actual Results:
Presenting view controller and its subviews become inverted colors.

Version:
iOS 11/12

Notes:
Attached sample code and recorded video. (https://github.com/ethanhuang13/Bug.IgnoreSmartInvert)

Configuration:

-
Product Version: iOS 11/12
Created: 2018-10-30T08:57:24.480660
Originated: 2018-10-30T16:53:00
Open Radar Link: http://www.openradar.me/45663655

**Comment by openradar-mirror** [comment]
Resolved: 30-Nov-2018
Modified: 2018-11-30T03:49:35.260633

**Comment by openradar-mirror** [comment]
Resolved: 30-Nov-2018
Modified: 2018-11-30T03:49:35.260633

**Comment by openradar-mirror** [comment]
Resolved: 30-Nov-2018
Modified: 2018-11-30T03:49:35.260633

**Comment by openradar-mirror** [comment]
Resolved: 30-Nov-2018
Modified: 2018-11-30T03:49:35.260633

**Comment by openradar-mirror** [comment]
Resolved: 30-Nov-2018
Modified: 2018-11-30T03:49:35.260633

**Comment by openradar-mirror** [comment]
Resolved: 30-Nov-2018
Modified: 2018-11-30T03:49:35.260633

**Comment by openradar-mirror** [comment]
Resolved: 30-Nov-2018
Modified: 2018-11-30T03:49:35.260633
